Koronavilkku/Koronavilkku/UI/Components/StatisticsCard.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+
+final class StatisticsCard: WideRowElement {
```

Comment:
Perhaps plain CardElement would suit better here?

Reply:
<img width="267" alt="Screenshot 2022-03-23 at 15 56 18" src="https://user-images.githubusercontent.com/17072260/159716094-591d5c17-2a3d-4830-a45f-f9ccee87710b.png">

This is how the view looks with CardElement

Reply:
I think it suits better.